### ISSUE TITLE
Audio stream titles don't align with video titles.

### ISSUE THREAD

**Issue by TheAznCoderPro** [crunchy-labs/crunchy-cli#384]
When using `archive` and merging multiple videos together, the audio and subtitle have a value stating `[Video: #<something>]`, however this number doesn't match with the actual video stream.
Command used: ```crunchy-cli archive 'https://www.crunchyroll.com/series/GYVD2K1WY/the-promised-neverland' -a en-US -a ja-JP```
**Expected behavior**
The video and audio streams match:
For example:
```
  Stream #0:4(en-US): Subtitle: ass (ssa) (forced)
        title           : en-US (CC) [Video: #1]
  Stream #0:5(en-US): Subtitle: ass (ssa)
        title           : en-US [Video: #2]
  Stream #0:6(es-419): Subtitle: ass (ssa)
        title           : es-419 [Video: #2]
  Stream #0:7(pt-BR): Subtitle: ass (ssa)
        title           : pt-BR [Video: #2]
```
**Actual outcome**
```
  Stream #0:4(en-US): Subtitle: ass (ssa) (forced)
        title           : en-US (CC) [Video: #1]
  Stream #0:5(en-US): Subtitle: ass (ssa)
        title           : en-US [Video: #2]
  Stream #0:6(es-419): Subtitle: ass (ssa)
        title           : es-419 [Video: #3]
  Stream #0:7(pt-BR): Subtitle: ass (ssa)
        title           : pt-BR [Video: #4]
```
Notice the `title` field of the subtitles goes up to 4 where as only 2 videos exist
Same issue occurs with audio streams.
Video tracks:
```
  Stream #0:0: Video: h264
        title           : #1
  Stream #0:1: Video: h264
        title           : #2
```

**Version used**
 - OS: MacOS
 - Version: `crunchy-cli 3.5.0 (8fff807 2024-04-19)`

**Comment by Frooastside** [comment]
I think the numbers are only there for the user to know which video and audio belong to each other as they video numbers are mentioned in the audio titles

**Comment by bytedream** [comment]
Thanks for reporting. Got fixed in [v3.5.1](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.1)
